lib/authlogic/acts_as_authentic/login.rb

```diff
@@ -152,10 +152,14 @@ def find_by_smart_case_login_field(login)
         private
 
           def find_with_case(field, value, sensitivity = true)
+            gem_version = Gem::Version.new(Rails.version)
```

Comment:
On second thought, maybe `activerecord_version`, `ar_version`, or `ar_gem_version` is better :)